[CS2113-T10-1] EasInternship

Ridiculouswifi opened the pull request:
EasInternship aims to streamline the process of searching for internship opportunities. It optimized for CLI users who are familiar with typing commands and general CLI usage.

TJW0911 commented on src/main/java/seedu/commands/AddCommand.java:
```diff
@@ -0,0 +1,83 @@
+        Internship newInternship = new Internship(role, company, startDate, endDate);
+        internships.addInternship(newInternship);
+        uiCommand.showEditedInternship(newInternship, "add");
+    }
```
This method is a bit long, perhaps you could refactor it to separate the argument parsing and the logic for creation of a new internship.

TJW0911 commented on src/main/java/seedu/commands/AddCommand.java:
```diff
@@ -0,0 +1,83 @@
+        for (String arg : args) {
+            String[] words = arg.split(" ", 2);
+            String flag = words[0];
+            switch (flag) {
+            case "role":
+                hasRole = true;
+                if (words.length > 1) {
+                    role = words[INDEX_DATA];
+                } else {
+                    uiCommand.addInvalidFlag(flag);
+                }
+                break;
+            case "company":
+                hasCompany = true;
+                if (words.length > 1) {
+                    company = words[INDEX_DATA];
+                } else {
+                    uiCommand.addInvalidFlag(flag);
+                }
+                break;
+            case "from":
+                if (words.length > 1) {
+                    startDate = words[INDEX_DATA];
+                } else {
+                    uiCommand.addInvalidFlag(flag);
+                }
+                break;
+            case "to":
+                if (words.length > 1) {
+                    endDate = words[INDEX_DATA];
+                } else {
+                    uiCommand.addInvalidFlag(flag);
+                }
+                break;
+            default:
+                uiCommand.addInvalidFlag(flag);
+                break;
+            }
```
Since the if checks and else statements are the same for all the switch cases, maybe you could do the if else checks before the switch cases so that the switch cases would be more concise.

TJW0911 commented on src/main/java/seedu/duke/Internship.java:
```diff
@@ -0,0 +1,129 @@
+     *
+     * @param userStatus user-inputted status.
+     */
+    public void updateStatus(String userStatus) throws InvalidStatus {
+        List<String> statuses = Arrays.asList("Application Pending", "Application Completed", "Accepted", "Rejected");
+        for (String status : statuses) {
+            if (status.equalsIgnoreCase(userStatus)) {
+                this.status = status;
+                return;
+            }
+        }
+        throw new InvalidStatus();
+    }
```
You could consider using an enum here to make the code easier to update and expand on in the future.

3CCLY commented on docs/DeveloperGuide.md:
```diff
@@ -8,8 +8,237 @@
+
+This is represented with the class diagram below:
+
+![](UML/Internship-Model_Component.png)
```
I believe the visibility of the methods and attributes in this diagram should be signs like "-" and "+" instead of squares and circles. You can consider configuring your plantuml to reflect this.

gskang-22 replied:
iirc we are supposed to change the default style of the class diagram that is given by PlantUML? Ie use "+" and "-" instead of red squares and green circles.

gskang-22 replied:
also can consider only adding the important methods, instead of spamming all your methods to improve clarity

gskang-22 commented on docs/DeveloperGuide.md:
```diff
@@ -8,8 +8,237 @@
+
+## 1. Design
+
+### 1.1 Architecture
```
Consider adding an architecture diagram

3CCLY commented on docs/DeveloperGuide.md:
```diff
@@ -8,8 +8,237 @@
+
+## 1. Design
+
+### 1.1 Architecture
+The architecture of EasInternship is designed to follow the MVC (Model-View-Controller) pattern to facilitate separation of concerns, modularity, and maintainability.
+
+- **View (UI)**: Responsible for interacting with the user by displaying output and reading input.
+- **Controller (Command and Parser)**: Responsible for parsing user input and invoking the appropriate commands.
+- **Model (InternshipList)**: Manages the state of the application, including the list of internships and tasks.
+- **Storage**: Responsible for loading and saving data from and to the disk.
+
```
I feel that an architecture diagram help could be a very helpful diagram to add here to give a brief overview of how your different classes interact with each other

3CCLY commented on docs/UML/loadFromFile.png:
Is the format for the "new Internship()" method call correct? My understanding is that it should call a constructor of a new object of "InternshipList" that is created at that point, instead of pointing to an existing object of "InternshipList" with a lifeline before the "new Internship()" method is called.

gskang-22 commented on docs/DeveloperGuide.md:
```diff
@@ -8,8 +8,237 @@
+There are two main functions, `loadFromFile` and `saveFromFile`.
+
+The following sequence diagrams depict how the Storage Functions work.
+![](UML/loadFromFile.png)
```
the sequence diagram shouldnt have 2 sets of class name?
![image](https://github.com/user-attachments/assets/d52df515-8d97-4f24-97aa-683852f80da6)

3CCLY commented on docs/DeveloperGuide.md:
```diff
@@ -8,8 +8,237 @@
+
+---
+
+### Overview
+
+The `EasInternship` class serves as the entry point of the application. It manages the application's main loop, where the user is continually prompted for input, and commands are processed in response. The class is responsible for initializing the UI, loading saved data, and handling user input until the user chooses to exit the program.
+
```
I feel that the overview would be more fitting being placed earlier in the documentation guide, perhaps you can give this some consideration?

gskang-22 commented on docs/DeveloperGuide.md:
```diff
@@ -8,8 +8,237 @@
+
+---
+
+### Overview
```
This structure feels abit weird to me. Might be better if you continued to follow the naming structure above? (ie 1.2, 1.3, 2.1, etc)

3CCLY reviewed (commented):
Quite a detailed documentation guide, good job on that

Ridiculouswifi committed:
Merge branch 'master' into branch-Deadline

Ridiculouswifi committed:
Add getUsage of CalendarCommand to HelpCommand

Ridiculouswifi committed:
Merge branch 'master' into branch-UiClass

Ridiculouswifi committed:
Refactor output of SortCommand

Ridiculouswifi committed:
Refactor HelpCommand, FAQ and Invalid inputs section in User Guide

Toby-Yu committed:
Merge pull request #113 from Ridiculouswifi/branch-UiClass

Refactor output of SortCommand

Ridiculouswifi committed:
Refactor FilterCommand and DeleteCommand Sections

Ridiculouswifi committed:
Add CalendarCommand section

Ridiculouswifi committed:
Refactor Update and Remove Command sections

Toby-Yu committed:
update the ug for stanard output for sortCommand

Toby-Yu committed:
Merge pull request #104 from Toby-Yu/Update-Value-proposition-and-user-stories

update the value proposition and user story

Ridiculouswifi committed:
Add List Command section

Toby-Yu committed:
update 1 in DG of user stories

Ridiculouswifi committed:
Edit Product Description

Ridiculouswifi committed:
Update DeveloperGuide.md

Toby-Yu committed:
Merge pull request #115 from Toby-Yu/update1-in-user-stories

update 1 in DG of user stories

Toby-Yu committed:
Merge pull request #112 from Ridiculouswifi/branch-Deadline

Add CalendarCommand getUsage to HelpCommand

jadenlimjc committed:
added assertions to internship and internshiplist

rahuljai-05 committed:
Resolve merge conflicts

# Conflicts:
#	src/main/java/seedu/duke/Internship.java
#	src/main/java/seedu/duke/Parser.java
#	src/main/java/seedu/duke/Storage.java

jadenlimjc committed:
edited plantUML diagrams

Toby-Yu committed:
Merge pull request #114 from Toby-Yu/Update-the-output-format-in-UG

update the ug for stanard output for sortCommand

rahuljai-05 committed:
Merge pull request #82 from rahuljai-05/Rahul-Favourites

Implement functionality to mark internships as favourite and filter by favourite internships

Ridiculouswifi committed:
Resolve merge conflict

Ridiculouswifi committed:
Revert to previous commit

Ridiculouswifi committed:
Resolve merge conflicts

Toby-Yu committed:
some code format

Toby-Yu committed:
Add sort by all field in the favourite internship list and add corresponding change in UG and DG for SortCommand

Toby-Yu committed:
change some name standard

Toby-Yu committed:
Merge pull request #105 from Ridiculouswifi/branch-UserGuide

Update User Guide

jadenlimjc committed:
fixed error with assertions

jadenlimjc committed:
merge with assertion branch

Toby-Yu committed:
Merge branch 'master' into add-sort-favouritei-in-sortCommand

Toby-Yu committed:
Merge pull request #118 from jadenlimjc/branch-updatePlantUML

updated plantUML diagrams to fix minor bugs

jadenlimjc committed:
fixed bug where TOCs was not working as expected

Toby-Yu committed:
Merge pull request #122 from Toby-Yu/add-sort-favouritei-in-sortCommand

Add sort by all field in the favourite internship list and add corres…

Toby-Yu committed:
Merge branch 'AY2425S1-CS2113-T10-1:master' into master

Toby-Yu committed:
add some assertion for the SortCommand and add sort company and sort company in favourite. Also modify UG and DG and update the sequence diagram for SortCommand

Toby-Yu committed:
change some code format

Toby-Yu committed:
remove one assertion

Toby-Yu committed:
Merge pull request #123 from Toby-Yu/add-sort-company-in-SortCommand

Add sort company and sort company in favourite in sort command, and add some assertion in sortCommand. Also update the UG ,DG and the sequence diagram for the SortCommand.

jadenlimjc committed:
updated DG

jadenlimjc committed:
Merge branch 'master' into branch-updateDG

jadenlimjc committed:
updated Storage to deal with manual edits of .txt file

jadenlimjc committed:
minor bugfixes to DG

jadenlimjc committed:
Merge branch 'master' into branch-updateStorage

jadenlimjc committed:
Merge pull request #125 from jadenlimjc/branch-updateDG

Branch update dg

Ridiculouswifi committed:
Add assertions to UpdateCommand

Ridiculouswifi committed:
Merge branch 'master' into branch-updateStatus

Ridiculouswifi committed:
Add assertion to updateStatus method

jadenlimjc committed:
updated Storage to skip over corrupted lines instead of deleting the entire file. also takes in blank field for skills.

jadenlimjc committed:
Merge pull request #127 from jadenlimjc/branch-updateStorage

Fix bug where .txt file is manually changed and storage is unable to load file

jadenlimjc committed:
removed bugged assertion

rahuljai-05 committed:
Fix bugs

1. Favourite internships not being loaded
2. Add command displaying invalid fields from previous prompt
3. Structure happy paths in InternshipList

rahuljai-05 committed:
Delete Duke placeholder file

Ridiculouswifi committed:
Merge branch 'master' into branch-updateStatus

Ridiculouswifi committed:
Set up simple logging system

Ridiculouswifi committed:
Refactor with proper variable names

Ridiculouswifi committed:
Fix checkstyle errors

Ridiculouswifi committed:
Add logging to StorageClass

Ridiculouswifi committed:
Remove unnecessary BotLogger class

Ridiculouswifi committed:
Remove Duke Class

Ridiculouswifi committed:
Set up to accept folder without Duke class

rahuljai-05 committed:
Clean up code

Convert invalid integer messages to exceptions
Remove tests which cannot be performed from the
user end

rahuljai-05 committed:
Merge pull request #128 from Ridiculouswifi/branch-updateStatus

Add Logging to Methods and Classes

rahuljai-05 committed:
Remove unused imports

Ridiculouswifi committed:
Add logging to InternshipList

Ridiculouswifi committed:
Add logging to Parser

Ridiculouswifi committed:
Add logging to Command Classes

Ridiculouswifi committed:
Merge branch 'master' into branch-UserGuide

rahuljai-05 committed:
Error message for repeated flags in FilterCommand

rahuljai-05 committed:
Merge pull request #131 from rahuljai-05/Rahul-BugFixes

Fixed bugs and refactored code to display clearer error messages

jadenlimjc committed:
Merge branch 'master' into branch-Assertions

Ridiculouswifi committed:
Merge branch 'master' into branch-updateStatus

Ridiculouswifi committed:
Fix merge conflicts

Ridiculouswifi committed:
Merge branch 'master' into branch-UserGuide

Ridiculouswifi committed:
Add Command Summary

rahuljai-05 committed:
Add FilterCommand and FavouriteCommand to UG

rahuljai-05 committed:
Fix bug with favourite command

Fixed a bug where an internship remains in the favouriteInternships list
even if it has already been deleted from the main internships list

jadenlimjc committed:
Update InternshipList.java

rahuljai-05 committed:
Merge pull request #116 from jadenlimjc/branch-Assertions

add authorship comments to Internshiplist

rahuljai-05 committed:
Add anchor link

jadenlimjc committed:
Merge pull request #132 from Ridiculouswifi/branch-updateStatus

Continue Adding Logging

jadenlimjc committed:
Merge pull request #133 from Ridiculouswifi/branch-UserGuide

Add Command Summary

jadenlimjc committed:
Merge pull request #137 from rahuljai-05/Rahul-UserGuide

Add commands to UG and fix FavouriteCommand bug

rahuljai-05 committed:
Add PPP

rahuljai-05 committed:
Handle invalid date inputs for AddCommand

rahuljai-05 committed:
Merge pull request #140 from rahuljai-05/Rahul-AddCommandBugFix

Handle invalid date inputs for AddCommand

rahuljai-05 committed:
Merge pull request #139 from rahuljai-05/Rahul-PPP

Add PPP version 1.0

rahuljai-05 committed:
Fix bug

Ridiculouswifi committed:
Merge pull request #141 from rahuljai-05/Rahul-UgBugFix

Toby-Yu committed:
make UG ,DG, and the actual usage of SortCommand is consistent. Also swap the position of sort and favourite in UG.

Toby-Yu committed:
Add slightly change in format of the sort in UG

Toby-Yu committed:
add Toby pic in about us page

Ridiculouswifi committed:
Update loading: Revert status when status is corrupted in data file

Ridiculouswifi committed:
Merge pull request #213 from Toby-Yu/Address-bug-for-PE-1

make UG ,DG, and the actual usage of SortCommand is consistent. Also …

Ridiculouswifi committed:
Merge pull request #214 from Toby-Yu/modify-UG-sortCommand-1

slightly modify some format of ug sort command

Ridiculouswifi committed:
Fix typo in invalid status message

jadenlimjc committed:
issue 210

jadenlimjc committed:
issue 185, issue 208, issue 146

Ridiculouswifi committed:
Update invalid deadline message to be clearer

jadenlimjc committed:
issue 202, issue 187

Ridiculouswifi committed:
Updating deadline ignores all inputs after a valid date

jadenlimjc committed:
updated all diagrams

jadenlimjc committed:
fixed dg related issues

rahuljai-05 committed:
Fix issue #180

Incorrect exception caught for filter -favourite

rahuljai-05 committed:
Remove static keywords

Ridiculouswifi committed:
Fix checkstyle errors

jadenlimjc committed:
issue 200

jadenlimjc committed:
issue 199, issue 197

jadenlimjc committed:
issue 179

Toby-Yu committed:
improve the sortCommand usage

jadenlimjc committed:
updated loadFromFile

jadenlimjc committed:
fixed checkstyle errors

Toby-Yu committed:
Change the sorting method of sort -status and sort -status in favourite , that sorted by status, (application pending, application completed, accepted, rejected) with this ascending order. Within the same status, the order is sorted by role.

Toby-Yu committed:
change the code format

jadenlimjc committed:
issue 172

jadenlimjc committed:
issue 170

jadenlimjc committed:
issue 165

jadenlimjc committed:
issue 159

Ridiculouswifi committed:
Date inputs are stricter

Ridiculouswifi committed:
Lenient dat parsing in the Internship class

Ridiculouswifi committed:
Edit getUsage messages for UpdateCommand and RemoveCommand

Ridiculouswifi committed:
Disallow duplicate skills

Ridiculouswifi committed:
Merge branch 'master' into branch-updateStatus

Ridiculouswifi committed:
Edit Update and Remove commands sections in User Guide

Ridiculouswifi committed:
Merge branch 'master' into branch-Deadline

Ridiculouswifi committed:
Add company name to calendar function output

Ridiculouswifi committed:
Separate print date function to shorten showCalendar method

Ridiculouswifi committed:
Update Calendar example in User Guide

Ridiculouswifi committed:
Merge branch 'master' into branch-DeveloperGuide

Ridiculouswifi committed:
Merge pull request #215 from Ridiculouswifi/branch-updateStatus

[Please Merge :)] Clear bugs from update function

Ridiculouswifi committed:
Merge pull request #218 from Toby-Yu/change-sortCommand-Usage

improve the sortCommand usage

Ridiculouswifi committed:
Merge pull request #223 from Ridiculouswifi/branch-Deadline

Improve Calendar Function

jadenlimjc committed:
fixed checkstyle issues

jadenlimjc committed:
fixed checkstyle errors

jadenlimjc committed:
Merge pull request #221 from jadenlimjc/branch-UGIssues

Branch ug issues

jadenlimjc committed:
Merge branch 'master' into branch-fixStorageIssues

Ridiculouswifi committed:
Edit UpdateCommand Sequence Diagram

Ridiculouswifi committed:
Merge branch 'master' into branch-DeveloperGuide

jadenlimjc committed:
fixed checkstyle errors

jadenlimjc committed:
Merge remote-tracking branch 'origin/branch-fixStorageIssues' into branch-fixStorageIssues

jadenlimjc committed:
fixed checkstyle errors

Ridiculouswifi committed:
Edit RemoveCommand Sequence Diagram

jadenlimjc committed:
changed incorrect detection to work in storage file instead

jadenlimjc committed:
issue 225

jadenlimjc committed:
fix checkstyle errors

rahuljai-05 committed:
Refactor code according to code quality guidelines

Renamed a few variables and added JavaDoc comments

rahuljai-05 committed:
Resolve merge conflicts

# Conflicts:
#	src/main/java/seedu/EasInternship/Internship.java
#	src/main/java/seedu/EasInternship/InternshipList.java
#	src/main/java/seedu/commands/Command.java
#	src/main/java/seedu/commands/FavouriteCommand.java
#	src/main/java/seedu/commands/UpdateCommand.java
#	src/main/java/seedu/duke/Parser.java

rahuljai-05 committed:
Fix error

rahuljai-05 committed:
Fix checkstyle errors

jadenlimjc committed:
Merge pull request #220 from jadenlimjc/branch-fixStorageIssues

Branch fix storage issues

rahuljai-05 committed:
Fix checkstyle errors

rahuljai-05 committed:
Resolve merge conflicts

jadenlimjc committed:
javaDoc comments for InternshipList

rahuljai-05 committed:
Wrap lines

rahuljai-05 committed:
Fix comment spacing

Ridiculouswifi committed:
Add CalendarCommand Section to DG

jadenlimjc committed:
javaDov comments for AddCommand and DeleteCommand

Toby-Yu committed:
update readme

jadenlimjc committed:
javaDoc comments for Deadline

Toby-Yu committed:
Merge pull request #227 from jadenlimjc/master

javaDoc comments for InternshipList

Ridiculouswifi committed:
Add acknowledgements and Edit links in the index

jadenlimjc committed:
javaDoc comments for Internship

rahuljai-05 committed:
Change main application location

jadenlimjc committed:
javaDoc comments for Storage

rahuljai-05 committed:
Resolve merge conflicts

# Conflicts:
#	src/main/java/seedu/EasInternship/InternshipList.java

jadenlimjc committed:
updated to fix checkstyle errors

Toby-Yu committed:
add the respo code comment

jadenlimjc committed:
Merge pull request #226 from rahuljai-05/Rahul-PEdryBugFixes

Refactor according to code quality guidelines

Toby-Yu committed:
remove the comment

Ridiculouswifi committed:
Add Ui Class Diagram to DG

Toby-Yu committed:
Merge branch 'master' into modify-ReadMe

Toby-Yu committed:
solve merge conflicts

Toby-Yu committed:
Merge pull request #228 from Toby-Yu/modify-ReadMe

update readme

rahuljai-05 committed:
Chnage Filter Command sequence diagram

Ridiculouswifi committed:
Edit return message for Remove deadline

Ridiculouswifi committed:
Merge branch 'master' into branch-updateStatus

Ridiculouswifi committed:
Merge branch 'master' into branch-DeveloperGuide

Ridiculouswifi committed:
Fix variable typo

Ridiculouswifi committed:
Merge branch 'master' into branch-updateStatus

Ridiculouswifi committed:
Refactor package name

Ridiculouswifi committed:
Merge pull request #224 from Ridiculouswifi/branch-DeveloperGuide

Edit Developer Guide

Ridiculouswifi committed:
Edit build.gradle file

Ridiculouswifi committed:
Merge branch 'master' into branch-PPP

jadenlimjc committed:
Merge branch 'master' of https://github.com/AY2425S1-CS2113-T10-1/tp into pr/229

jadenlimjc committed:
updated ReadME to change link to PPP

Toby-Yu committed:
update sortCommandtest and toby PPP

Toby-Yu committed:
change to pass the auto test

jadenlimjc committed:
Merge pull request #232 from jadenlimjc/branch-updateUG

updated ReadME to change link to PPP

jadenlimjc committed:
updated PPP PR linkes

jadenlimjc committed:
Merge branch 'AY2425S1-CS2113-T10-1:master' into branch-updateUG

jadenlimjc committed:
Merge pull request #234 from jadenlimjc/branch-updateUG

Branch update ug

jadenlimjc committed:
Merge pull request #233 from Toby-Yu/update-sortCommandtest-Junit-test

update sortCommandtest and toby PPP

Ridiculouswifi committed:
Update PPP

Ridiculouswifi committed:
Merge branch 'master' of https://github.com/Ridiculouswifi/tp into branch-updateStatus

Ridiculouswifi committed:
Merge pull request #231 from Ridiculouswifi/branch-updateStatus

Edit message to Remove deadline

Ridiculouswifi committed:
Edit README page

Ridiculouswifi committed:
Standardise PPP Overview

Ridiculouswifi committed:
Merge branch 'master' into branch-PPP

Ridiculouswifi committed:
Merge pull request #235 from Ridiculouswifi/branch-PPP

Update PPP

rahuljai-05 committed:
Add FavouriteCommand to DG

Ridiculouswifi committed:
Merge branch 'master' into branch-PPP

jadenlimjc committed:
added fullstops

rahuljai-05 committed:
Resolve merge conflicts

# Conflicts:
#	docs/DeveloperGuide.md

Ridiculouswifi committed:
Add full stops to ridiculouswifi PPP

Ridiculouswifi committed:
Fix typo in README

Ridiculouswifi committed:
Merge pull request #237 from Ridiculouswifi/branch-PPP

Fix minor typos to ridiculouswifi PPP

jadenlimjc committed:
Merge pull request #238 from jadenlimjc/master

added fullstops

jadenlimjc committed:
Merge pull request #230 from rahuljai-05/Rahul-FilterCommandDiagramUpdate

Change Filter Command sequence diagram

rahuljai-05 committed:
Add Sequence Diagram for FavouriteCommand

rahuljai-05 committed:
Resolve merge conflicts

# Conflicts:
#	docs/DeveloperGuide.md

rahuljai-05 committed:
Add sequence diagram image to DG

Ridiculouswifi committed:
Add link to PPP in AboutUs

jadenlimjc committed:
Merge pull request #236 from rahuljai-05/Rahul-FavouriteCommandDG

Add FavouriteCommand to DG

Ridiculouswifi committed:
Add personal photo ridiculouswifi

Ridiculouswifi committed:
Merge pull request #239 from Ridiculouswifi/master

Edit AboutUs

jadenlimjc committed:
updated seq diagram return statements

jadenlimjc committed:
Merge pull request #240 from jadenlimjc/master

updated seq diagram return statements

rahuljai-05 committed:
edit PPP

rahuljai-05 committed:
Remove lines from UG

rahuljai-05 committed:
Remove commit hash from DG

# Conflicts:
#	docs/DeveloperGuide.md

rahuljai-05 committed:
Merge pull request #241 from rahuljai-05/Rahul-PPPv2.0

Rahul PPP v2.0

rahuljai-05 committed:
Add FilterCommand and FavouriteCommand to DG index

Fix formattinga at the end

rahuljai-05 committed:
Put FilterCommand sequence diagram after text

rahuljai-05 committed:
Add info to AboutUs page

jadenlimjc committed:
Merge pull request #242 from rahuljai-05/Rahul-DGBugFix

Add FilterCommand and FavouriteCommand to DG index

Ridiculouswifi committed:
Merge branch 'master' into branch-UserGuide

jadenlimjc committed:
fix sortDeadline bug

jadenlimjc committed:
remove unused imports

jadenlimjc committed:
Merge pull request #243 from jadenlimjc/branch-fixSortDeadline

Branch fix sort deadline

Ridiculouswifi committed:
Edit output until sort in UG

Ridiculouswifi committed:
Edit build.gradel file

Ridiculouswifi committed:
Merge branch 'master' into branch-UserGuide

jadenlimjc committed:
updated rahul ppp link

Ridiculouswifi committed:
Fix example outputs in UG

jadenlimjc committed:
Merge pull request #244 from jadenlimjc/master

updated rahul ppp link

Ridiculouswifi committed:
Add link to release in UG

Ridiculouswifi committed:
Merge pull request #245 from Ridiculouswifi/branch-UserGuide

Update UG

Ridiculouswifi committed:
Expand on FAQ in UG

Ridiculouswifi committed:
Merge pull request #246 from Ridiculouswifi/branch-UserGuide

Expand on FAQ in UG

Ridiculouswifi committed:
Merge branch 'master' into branch-DeveloperGuide

Ridiculouswifi committed:
Fix formatting errors in DG

Ridiculouswifi committed:
Merge pull request #247 from Ridiculouswifi/branch-DeveloperGuide

Fix errors in DG

Ridiculouswifi committed:
Merge branch 'master' into branch-Deadline

Ridiculouswifi committed:
Fix sort by deadline

Ridiculouswifi committed:
Merge pull request #248 from Ridiculouswifi/branch-Deadline

Fix sort by deadline

Ridiculouswifi committed:
Fix Sequence Diagram in DG

Ridiculouswifi committed:
Fix Sequence Diagram in DG

Ridiculouswifi committed:
Fix Sequence Diagram in DG

Ridiculouswifi committed:
Update jar link in UG
